website/cue/reference/releases/0.22.0.cue

```diff
@@ -0,0 +1,767 @@
+				consume journald events from a remote system. This matches the Debian package.
+				"""
+			pr_numbers: [12563]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now allows configuration of `extra_namespace_label_selector` which Vector
+				will to use select the pods to capture the logs of, if set, based on labels attached to the pod
+				namespace. This is similar to the `extra_label_selector` option which applies to pod labels.
+				"""
+			contributors: ["@anapsix"]
+			pr_numbers: [12438]
+		},
+		{
+			type: "enhancement"
+			scopes: ["kubernetes_logs source"]
+			description: """
+				The `kubernetes_logs` source now reads events in order whenever a pod log file rotates. Previously Vector could start reading the new file before it finished processing the previous one, resulting in the logs being out-of-order.
+				"""
+			pr_numbers: [12330]
+			contributors: ["@sillent"]
+		},
+		{
+			type: "enhancement"
+			scopes: ["vrl"]
+			description: """
+				The `parse_json` function now takes an optional `max_depth` parameter to control how far it will recurse when deserializing the event. Once the depth limit is hit, the remainder of the fields is left as raw JSON in the deserialized event.
+
+				For example:
+
+				```coffeescript
+				parse_json!("{\"1\": {\"2\": {\"3\": {\"4\": {\"5\": {\"6\": \"finish\"}}}}}}", max_depth: 5)
+				```
+
+				Yields:
+
+				```json
+				{ "1": { "2": { "3": { "4": { "5": "{\"6\": \"finish\"}" } } } } }
+				```
+
+				The default remains no max depth limit.
+				"""
+			pr_numbers: [12545]
+			contributors: ["@nabokihms"]
+		},
+		{
+			type: "enhancement"
+			scopes: ["observability"]
+			description: """
+				A new `component_received_events_count` histogram metric was added to record the sizes of event batches
+				passed around in Vector's internal topology. Note that this is different than sink-level batching. It is
+				mostly useful for debugging performance issues in Vector due to low internal batching.
+				"""
+			pr_numbers: [11290]
+		},
+		{
+			type: "fix"
+			scopes: ["vrl"]
+			description: """
+				The `log` function in VRL no longer wraps logged string values in quotes. This was causing double
+				quoting for sink encodings like `json`.
+				"""
+			pr_numbers: [12609]
+			contributors: ["@nabokihms"]
+		},
+		{
+			type: "fix"
+			scopes: ["aws_s3 source"]
+			description: """
+				The `aws_s3` source now handles S3 object keys that contain spaces. Previously Vector would encounter
+				a 404 when querying for objects due to not decoding spaces correctly from the SQS object notification.
+				"""
+			pr_numbers: [12664]
+		},
+		{
+			type: "fix"
+			scopes: ["gcp provider"]
+			description: """
+				GCP sinks now correctly handle authentication token refreshing from the metadata service.
+				"""
+			pr_numbers: [12645]
+		},
```

Comment:
It might be worth clarifying this is only relevant to when the healthcheck fails.